transformer_lens/loading_from_pretrained.py

```diff
@@ -1665,6 +1709,79 @@ def convert_qwen_weights(qwen, cfg: HookedTransformerConfig):
     return state_dict
 
 
+def convert_qwen2_weights(qwen, cfg: HookedTransformerConfig):
```

Comment:
Can you rename this to qwen1_5? I know Qwen2 this is consistent with HuggingFace, but IMO it's best to be consistent with the actual model names a user would recognise